src/test/lib/operators/operator_deep_copy_test.cpp

```diff
@@ -237,4 +249,57 @@ TEST_F(OperatorDeepCopyTest, Subquery) {
   EXPECT_TABLE_EQ_UNORDERED(copied_result, expected_copied);
 }
 
+TEST_F(OperatorDeepCopyTest, DeduplicationAmongRootAndSubqueryPQPs) {
+  // Uncorrelated subquery shares part of the root PQP (compare TPC-H Q11)
+  auto get_table_a = std::make_shared<GetTable>(_table_name_a);
+  auto get_table_b = std::make_shared<GetTable>(_table_name_b);
```

Comment:
I first thought that the comment related to the two lines below. Maybe, `in this test, we...`?